execution.rst

```diff
@@ -332,6 +332,39 @@ should be executed, then use the following command:
 Execution hooks can also be filtered based on tags. 
 See :ref:`filtering hooks with tags <filtering_hooks_with_tags>` for more information.
 
+Multiline tags
+^^^^^^^^^^^^^^
+
+In a specification, tags can be defined in multiple lines
+
+Example:
+
+.. code-block:: gauge
+    :linenos:
+
+    # Search Specification
+
+    The admin user must be able to search for available products on the search page.
+
+    Tags: search,
```

Comment:
this could be more specific if the tag lines are indicated via a "comma" or the indentation takes care of it.

Reply:
"Comma" is for multiple tags. Hence added the note that indentation is required specify tags in multiple lines.